pkg/lib/authn/sso/github.go

```diff
@@ -63,18 +63,19 @@ func (g *GithubImpl) NonOpenIDConnectGetAuthInfo(r OAuthAuthorizationResponse, _
 
 	idJSONNumber, _ := userProfile["id"].(json.Number)
 	email, _ := userProfile["email"].(string)
-	name, _ := userProfile["name"].(string)
+	login, _ := userProfile["login"].(string)
 	picture, _ := userProfile["avatar_url"].(string)
 	profile, _ := userProfile["html_url"].(string)
 
 	id := string(idJSONNumber)
 
 	authInfo.ProviderUserID = id
 	stdAttrs, err := stdattrs.Extract(map[string]interface{}{
-		stdattrs.Email:   email,
-		stdattrs.Name:    name,
-		stdattrs.Picture: picture,
-		stdattrs.Profile: profile,
+		stdattrs.Email:     email,
+		stdattrs.Name:      login,
+		stdattrs.GivenName: login,
```

Comment:
Using github username as the name looks good to me, but given name should be the first name of the user seems using username is not accurate?

Reply:
This is because given_name is default read-write and name is default hidden https://github.com/authgear/authgear-server/blob/master/pkg/lib/config/user_profile.go#L146
If we do not populate the github username as given_name, a new user signed up with github does not have visible "name" by default.

Reply:
I agree with using the github username in given_name for the reason @louischan-oursky mentioned.

If there is no other problem, shall we merge this and deploy to staging?